app/views/questions/index.html.erb

```diff
@@ -47,18 +46,74 @@
             <% if question["answers"][0] %>
               <%= question["answers"][0]["text"] %>
             <% end %>
-
           </div>
         </div>
-
-      </div>
       </div>
+    </div>
       <% collapse_count += 1 %>
-    <% end %>
+      <% end %>
+    </div>
+  <br>
   </div>
 
+                <!-- Defines how the modal will look -->
+<div class="modal fade" id="question_display_Modal" role="dialog" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <!-- Close button -->
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <!-- Displays the question at the top of the modal. Javascript finds the "modal-title" class and populates it with the question contents. -->
+        <h4 class="modal-title"></h4>
+      </div>
+      <div class="modal-body">
+        <!-- Creates the text box for the user to input their answer. The submit button is only able to be clicked once the user has some content in the text box. -->
+        <form accept-charset="UTF-8" action="/questions/question["id"]/answers" class="new_answer" data-remote="true" id="new_answer" method="post"><div style="display:none"><input name="utf8" type="hidden" value="✓"></div>
```

Comment:
This is another super-long line (as is the comment above it) that would be easier to scan if it was broken down / spanned multiple lines of code.